src/swagger/accomodation.doc.js

```diff
@@ -0,0 +1,192 @@
+// eslint-disable-next-line import/prefer-default-export
+export const accomodations = {
+  '/api/accommodations/': {
+    post: {
+      tags: ['Accommodations CRUD Operations'],
```

Comment:
May I suggest naming this tag `Accommodation (Admin only)` or just `Acccommodation (Admin)` as an alternative.